build.gradle

```diff
@@ -392,7 +399,7 @@ import com.google.gson.GsonBuilder
 
 final class JsonOrderStep {
     private JsonOrderStep() {}
-    private static final Gson GSON = new GsonBuilder().disableHtmlEscaping().setLenient().setPrettyPrinting().create();
+    private static final Gson GSON = new GsonBuilder().disableHtmlEscaping().setLenient().setPrettyPrinting().create()
```

Comment:
Why do you remove the semicolons? :(

Reply:
intellij alt-enter has its own mind. it's groovy it doeesnt even need semicolons!